scripts/deploy/troubleshooting/get-all-logs

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+function usage() {
+  echo "Usage: $0 <namespace> [since]"
+  echo
+  echo "This script saves logs from all pods within the specified namespace into local files and then creates a tar.gz archive of these logs."
```

Comment:
I think it's useful to mention here that if a pod crashed, its logs will _not_ be retrieved. 
So people can't, unfortunately, use this to debug failing pods. 

I think this script can also pessimistically always try to get the logs from the previously failed pod as well as the current one.

Reply:
Makes sense. Added.